tests/jq.test

```diff
@@ -2000,8 +2000,8 @@ null
 2
 
 .[] |= try tonumber
-["1", "2a", "3", 4]
-[1, 3, 4]
+["1", "2a", "3", " 4 ", "5.67", ".89", "-876", "+5.43", 21]
+[1, 3, 5.67, 0.89, -876, 5.43, 21]
```

Comment:
Note that spaces before (but not after) the number are still accepted in `--disable-decnum` builds